source/leuven/tier2_hardware/genius_hardware.rst

```diff
@@ -1,10 +1,11 @@
 Genius hardware
 ===============
 
-Genius is KU Leuven/UHasselt's most recent Tier-2 cluster. It has thin nodes, large memory nodes, as well as GPGPU nodes.
+Genius is one of KU Leuven/UHasselt's Tier-2 clusters, which is in production
+since 2018. It has thin nodes, large memory nodes, as well as GPGPU nodes.
```

Comment:
```suggestion
since 2018. It has thin nodes, large memory nodes, as well as GPU nodes.
```
(see related comments for the reasoning behind the suggestion)